lib/bindings/python/tests/test_aic_capacity.py

```
@@ -0,0 +1,86 @@
+
+    def _get_memory_usage(self, *_args, **_kwargs):
+        return self._memory
+
+
+class FakeModel:
+    def get_kvcache_bytes_per_sequence(self, seq_len):
+        return seq_len * _GIB
+
+
+class FakeDatabase:
+    system_spec = {"gpu": {"mem_capacity": 1000 * _GIB}}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)